gallery/visualtests/nxCombobox.js

```diff
@@ -271,5 +296,24 @@ describe('NxCombobox', function() {
       await input.focus();
       expect(await isVisible(dropdownMenu)).toBe(true);
     });
+
+    it('reopens dropdown after selection when clicking back into the input', async function() {
+      const inputSelector = `${basicExampleSelector} .nx-combobox__input input`,
+          dropdownMenuSelector = `${basicExampleSelector} .nx-dropdown-menu`,
+          buttonSelector = `${dropdownMenuSelector} .nx-dropdown-button:first-child`,
+          example = '#nx-combobox-basic-example',
+          [input, dropdownMenu, firstOptBtn, basicExample] = await waitAndGetElements(
+              inputSelector, dropdownMenuSelector, buttonSelector, example);
+
+      await input.click();
+      await firstOptBtn.click();
+      expect(await isVisible(dropdownMenu)).toBe(false);
+
+      await basicExample.click();
+      expect(await isFocused(input)).toBe(false);
+      await input.click();
```

Comment:
Crucially, we want to test that click reopens it _even if it never lost focus in the first place_.  ie. what you have here but without clicking the `basicExample` in between

Reply:
> Crucially, we want to test that click reopens it _even if it never lost focus in the first place_. ie. what you have here but without clicking the `basicExample` in between

I missed actually implementing the behaviour of re-opening the dropdown on click of the input, so that has been added here along with expanding the test: [1e7a194](https://github.com/sonatype/sonatype-react-shared-components/pull/876/commits/1e7a1947a39353cf05e7ba45124d34d0d28e7cdf)